droidlet/lowlevel/locobot/remote/slam_service.py

```diff
@@ -91,7 +92,7 @@ def add_obstacle(self, location, in_map=False):
         self.map_builder.add_obstacle(location)
 
     def update_map(self):
-        pcd = self.robot.get_current_pcd()[0]
+        pcd = safe_call(self.robot.get_current_pcd)[0]
```

Comment:
What is safe_cal for? why is this necessary?

Reply:
needed for making pyro4 fail-fast in case of a downstream service error (instead of waiting to timeout).
All the safe_call business will go away once we move to zerorpc.